spec.html

```diff
@@ -23527,7 +23527,8 @@
     <!-- es6num="19.3.3" -->
     <emu-clause id="sec-properties-of-the-boolean-prototype-object">
       <h1>Properties of the Boolean Prototype Object</h1>
-      <p>The Boolean prototype object is the intrinsic object %BooleanPrototype%. The Boolean prototype object is an ordinary object. It is not a Boolean instance and does not have a [[BooleanData]] internal slot.</p>
```

Comment:
> "The Boolean prototype object is itself a Boolean object, it has a [[BooleanData]] internal slot with the value false."

That definition is okay, because Boolean objects are also ordinary objects. It's just important to specify that `Boolean.prototype` is an ordinary object, otherwise implementations could redefine one of the internal methods of `Boolean.prototype`.